phoonnx/model_manager.py

```
@@ -0,0 +1,293 @@
+                    voice_id="piper_" + v["key"],
+                    lang=standardize_lang_tag(v["key"].split("-")[0]),
+                    model_url=base + [a for a in v["files"] if a.endswith(".onnx")][0],
+                    config_url=base + [a for a in v["files"] if a.endswith(".json")][0],
+                )
+                self.add_voice(voice)
+            except Exception:
+                print(f"Failed to get voice info for {v['key']}")
+
+    def get_mimic3_voice_list(self):
+        voice_list = "https://raw.githubusercontent.com/MycroftAI/mimic3/refs/heads/master/mimic3_tts/voices.json"
+        mimic3_voices = requests.get(voice_list, timeout=30).json()
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)